cli/node/mod.rs

```diff
@@ -370,18 +373,9 @@ pub fn node_resolve(
       let split_specifier = url.as_str().split(':');
       let specifier = split_specifier.skip(1).collect::<String>();
 
-      // NOTE(bartlomieju): this will force `ProcState` to use Node.js polyfill for
-      // `module` from `ext/node/`.
-      if specifier == "module" {
-        return Ok(Some(ResolveResponse::Esm(
-          Url::parse("node:module").unwrap(),
-        )));
-      }
-
-      if let Some(resolved) = try_resolve_builtin_module(&specifier) {
-        return Ok(Some(ResolveResponse::Esm(resolved)));
-      } else {
-        return Err(generic_error(format!("Unknown module {}", specifier)));
+      if is_builtin_node_module(&specifier) {
+        let url = Url::parse(&format!("node:{}", specifier)).unwrap();
+        return Ok(Some(ResolveResponse::Esm(url)));
```

Comment:
I feel we should have `ResolveResponse::Builtin` variant

Reply:
I was thinking about refactoring this code earlier to use a different enum that's not deno_graph based, but then I forgot. IIRC, I don't believe this needs to use that `ResolveResponse` enum.